app/templates/events/import.hbs

```diff
@@ -1 +1,2 @@
-import data
+{{events/import-event}}
```

Comment:
This name doesn't exactly sound like a _component_. Lets call these two 
`events/event-import-section` and `events/imports-history-section`